### ISSUE TITLE
generate_lyric项目中target和logits形状

### ISSUE THREAD

**Issue by BellaZbl** [NELSONZHAO/zhihu#31]
1. cell = tf.contrib.rnn.MultiRNNCell([lstm])  这里是只有一层rnn吗？

2. logits = tf.contrib.layers.fully_connected(outputs, vocab_size, activation_fn=None)
这里logits的shape是  batch_size，seq_len，vocab_size可是target是batch_size，seq_len,embed_dim 请问这两个怎么求损失？
